fetch/content-type/resources/script-content-types.json

```diff
@@ -0,0 +1,87 @@
+[
+  {
+    "input": ["text/javascript;charset=windows-1252"],
+    "executes": true,
+    "encoding": "windows-1252"
+  },
+  {
+    "input": ["text/javascript;\";charset=windows-1252"],
+    "executes": true,
+    "encoding": "windows-1252"
+  },
+  {
+    "input": ["text/javascript\u000C"],
+    "executes": false,
+    "encoding": null
+  },
+  {
+    "input": ["\"text/javascript\""],
+    "executes": false,
+    "encoding": null
+  },
+  {
+    "input": ["text / javascript"],
```

Comment:
Suggest splitting this in two, one with the space in each location.

Also, should the cases be checked (instead or additionally) in wpt/mimesniff/mime-types/resources/mime-types.json?